deployment/terraform/assets/cluster.tf

```diff
@@ -271,7 +271,7 @@ EOF
 
 
 resource "aws_rds_cluster_instance" "cluster_instances" {
-  count                      = var.app_instance_count > 0 ? var.db_instance_count : 0
```

Comment:
I think I should check here if we have ExternalDBHost, and then, not run the RDS instances at all, instead of doing it by driver.